### DISCUSSION TITLE
Taskbar height and icon size mod failed on win11 26085

### DISCUSSION THREAD

**Discussion by digiant** [ramensoftware/windhawk-mods#573] [Q&A] [answered]
Hi all, it seems this mod doesn't support windows 11 insider preview build 26085, here is the log:

13:47:58.545 2608 explorer.exe  [WH] [taskbar-icon-size] [864:HookSymbols]: Optional symbol 5 doesn't exist
13:47:58.545 2608 explorer.exe  [WH] [taskbar-icon-size] [860:HookSymbols]: Unresolved symbol: 9

[taskbar icon size.log](https://github.com/ramensoftware/windhawk/files/14739384/taskbar.icon.size.log)

**Comment by m417z** [comment, marked as answer]
Hi, it was already reported here: https://github.com/ramensoftware/windhawk-mods/issues/562
